src/OpcUaStackServer/ServiceSet/MonitorManager.cpp

```diff
@@ -582,8 +582,9 @@ namespace OpcUaStackServer
 		}
 
 		// check monitored item list
+		monitoredItemIds_.erase(it1);
 		if (newMonitoredItemIds.size() > 0) {
```

Comment:
I'm sure that this functional worked correctly. But judging the history of file it could never work at all.

Reply:
I've understood. The problem is only for duplication monitored items inside of single subscription. I tested my application with different clients.